api/src/Responder/Responder/ApiResponder.php

```diff
@@ -109,6 +109,19 @@ public function createdItem($item, array $parameters = [])
     }
 
     /**
+     * Respond with a redirect.
+     *
+     * @param  string  $url
+     * @param  int     $status
+     * @param  array   $headers
+     * @return Response
+     */
+    public function redirect($url, $status = 302, array $headers = [])
+    {
+        return $this->getRedirectResponse($url, $status, $headers);
+    }
+
+    /**
```

Comment:
Just for your info, @Redjik is going to change the `BaseResponder` to extend `Response` directly so we can use those inherited methods directly for redirects. This won't affect this PR, but keep it in mind as a minor refactor will be needed once that is completed.